### PR TITLE
Billing Page - Displays

### PR DESCRIPTION
**Pull request by kelfro**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: frontend/src/pages/Landingpage.tsx [hunk 1]
```diff
@@ -136,6 +136,7 @@ const Landing = (props: RouteComponentProps) => {
           </a>
         </Footer>
       </Container>
+	  <div> { Billing } </div>
```
[the remark this answers is not in the document]
Comment (kelfro) [reply frontend/src/pages/Landingpage.tsx]: Yeah, I'm taking this out. I think at some point I was testing it and forgot I left it there.

File: frontend/src/pages/Billing.tsx [hunk 2]
```diff
@@ -1,6 +1,56 @@
...
-type Props = {};
-export const Button = () => {
-  return <div />;
+const Main = styled('div')`
+	margin: 50px
+`
+
+function Billing(props: RouteComponentProps) {
+	
+	const [info, setValues] = useState({
```
[the remark this answers is not in the document]
Comment (kelfro) [reply frontend/src/pages/Billing.tsx]: Changing this now. :)

File: frontend/src/pages/Billing.tsx [hunk 3]
```diff
@@ -1,6 +1,56 @@
...
+		});
+	};
+
+	return(
+		<>
+		<Main>
+			<div>
+			{/*Create form for collecting billing information - CC#, Exp. CVV [submit button]*/}
+			<form onSubmit={ printValues }>
+				<label>
```
[the remark this answers is not in the document]
Comment (kelfro) [reply frontend/src/pages/Billing.tsx]: I added a description for each label element.

File: frontend/tsconfig.json [hunk 4]
```diff
@@ -13,7 +13,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "preserve"
+    "jsx": "preserve",
+	"noImplicitAny": false
```
[the remark this answers is not in the document]
Comment (kelfro) [reply frontend/tsconfig.json]: I fixed this part by adding event types to the handlers and reverted tsconfig.json back to its original form so this should be all good.

**Comment by NandoTheessen** [comment]
[accordion menu](https://trello.com/c/mI60vYCD/45-accordion-menu)

**Comment by NandoTheessen** [comment]
Closing this as it is stale.

**Pull request closed without merging**